partcad/src/partcad/runtime_python_conda.py

```
@@ -64,57 +63,56 @@ async def once_async(self):
+                                "--json",
+                                "-p",
+                                self.path,
+                                "python==%s" % self.version if self.is_mamba else "python=%s" % self.version,
+                            ],
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            shell=False,
+                            encoding="utf-8",
+                        )
+                        _, stderr = p.communicate()
+                        if not stderr is None and stderr.strip() != "":
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)